homeassistant/components/zha/api.py

```diff
@@ -466,7 +517,10 @@ def async_load_api(hass):
         command = service.data.get(ATTR_COMMAND)
         command_type = service.data.get(ATTR_COMMAND_TYPE)
         args = service.data.get(ATTR_ARGS)
-        manufacturer = service.data.get(ATTR_MANUFACTURER) or None
+        manufacturer = None
+        #  only use manufacturer code for manufacturer clusters
+        if cluster_id >= 0xfc00:
```

Comment:
Should such values be extracted in constants?

Reply:
Sure